cmd/bee/cmd/start.go

```diff
@@ -26,25 +26,26 @@ import (
 func (c *command) initStartCmd() (err error) {
 
 	const (
-		optionNameDataDir            = "data-dir"
-		optionNameDBCapacity         = "db-capacity"
-		optionNamePassword           = "password"
-		optionNamePasswordFile       = "password-file"
-		optionNameAPIAddr            = "api-addr"
-		optionNameP2PAddr            = "p2p-addr"
-		optionNameNATAddr            = "nat-addr"
-		optionNameP2PWSEnable        = "p2p-ws-enable"
-		optionNameP2PQUICEnable      = "p2p-quic-enable"
-		optionNameDebugAPIEnable     = "debug-api-enable"
-		optionNameDebugAPIAddr       = "debug-api-addr"
-		optionNameBootnodes          = "bootnode"
-		optionNameNetworkID          = "network-id"
-		optionWelcomeMessage         = "welcome-message"
-		optionCORSAllowedOrigins     = "cors-allowed-origins"
-		optionNameTracingEnabled     = "tracing-enable"
-		optionNameTracingEndpoint    = "tracing-endpoint"
-		optionNameTracingServiceName = "tracing-service-name"
-		optionNameVerbosity          = "verbosity"
+		optionNameDataDir             = "data-dir"
+		optionNameDBCapacity          = "db-capacity"
+		optionNamePassword            = "password"
+		optionNamePasswordFile        = "password-file"
+		optionNameAPIAddr             = "api-addr"
+		optionNameP2PAddr             = "p2p-addr"
+		optionNameNATAddr             = "nat-addr"
+		optionNameP2PWSEnable         = "p2p-ws-enable"
+		optionNameP2PQUICEnable       = "p2p-quic-enable"
+		optionNameDebugAPIEnable      = "debug-api-enable"
+		optionNameDebugAPIAddr        = "debug-api-addr"
+		optionNameBootnodes           = "bootnode"
+		optionNameNetworkID           = "network-id"
+		optionWelcomeMessage          = "welcome-message"
+		optionCORSAllowedOrigins      = "cors-allowed-origins"
+		optionNameTracingEnabled      = "tracing-enable"
+		optionNameTracingEndpoint     = "tracing-endpoint"
+		optionNameTracingServiceName  = "tracing-service-name"
+		optionNameVerbosity           = "verbosity"
+		optionNameDisconnectThreshold = "disconnect-threshold"
```

Comment:
because of the code reformatting. `optionNameDisconnectThreshold ` is one character longer than the previous longest name, therefore all the `=` signs moved by one character.